Jenkinsfile

```diff
@@ -4,10 +4,10 @@
  */
 buildPlugin(useContainerAgent: true, configurations: [
   // Test the common case (i.e., a recent LTS release) on both Linux and Windows.
-  [ platform: 'linux', jdk: '11', jenkins: '2.361.4' ],
-  [ platform: 'windows', jdk: '11', jenkins: '2.361.4' ],
```

Comment:
"recent LTS release" was there for a reason ;)

Did something not work if you just bumped it, same way as below?

Reply:
> "recent LTS release" was there for a reason ;)
> 
> Did something not work if you just bumped it, same way as below?

1. yes there was compiling exceptions in java
2. I need latest LTS (current one) to integrate some ideas rom data-tables-api.
3. No body need to change jenkins version in 2 different files (this one + pom.xml) which leads to less mistakes.

Therefore I decide it will be fine to test it in this way also in future.